src/registrar/config/settings.py

```diff
@@ -357,13 +357,18 @@
 # and inline with a nonce, as well as allowing connections back to their domain.
 # Note: If needed, we can embed chart.js instead of using the CDN
 CSP_DEFAULT_SRC = ("'self'",)
-CSP_STYLE_SRC = ["'self'", "https://www.ssa.gov/accessibility/andi/andi.css"]
+CSP_STYLE_SRC = [
+    "'self'",
+    "https://www.ssa.gov/accessibility/andi/andi.css",
+    "https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css",
```

Comment:
I assume there was no other easy way to do this? Adding another hole in our CSP is not something we should do lightly. If somehow that version alias of select2 were to get replaced with something malicious anybody who accesses a view that loads scripts from this source could be vulnerable.

If it's going to be super expensive to replace this then it may be worth it, but I want to make sure it's something we're considering carefully.